lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -32,6 +35,13 @@
           stroke: #02A6D9;
         }
       }
+      &:hover {
+        + .nx-radio__circle {
+          #{$focus-border-selector} {
+            stroke: #045dd1;
```

Comment:
I couldn't see why we'd want to use the black border when hovering on a selected radio button and @dturner32 wasn't here to ask so I made it the selected blue color. It's a trivial change if it's wrong.

Reply:
> I couldn't see why we'd want to use the black border when hovering on a selected radio button and @dturner32 wasn't here to ask so I made it the selected blue color. It's a trivial change if it's wrong.

That does mean that hovering over a non-focused, checked radio has no visual effect.  Which is probably ok as clicking on it would also have no effect since it's already selected.

Reply:
> > I couldn't see why we'd want to use the black border when hovering on a selected radio button and @dturner32 wasn't here to ask so I made it the selected blue color. It's a trivial change if it's wrong.
> 
> That does mean that hovering over a non-focused, checked radio has no visual effect. Which is probably ok as clicking on it would also have no effect since it's already selected.

That was my thinking yes.

Reply:
I am ok with this adjustment